Eslint configuration

tiago154 opened the pull request:
#89 

Hi. I added eslint and put some rules according to the project.

So to validate the errors, run `npm run lint`

There is also (`npm run lint:fix`), where it is possible to apply corrections automatically.

ROODAY commented:
@tiago154 Thanks for the PR! One request though, can you resubmit this using yarn? We don't want to have both a package-lock and a yarn-lock file. Thanks!

tiago154 commented:
> @tiago154 Thanks for the PR! One request though, can you resubmit this using yarn? We don't want to have both a package-lock and a yarn-lock file. Thanks!

@ROODAY Of course! I removed the package-lock.json and updated the yarn.lock
